Update Gemfile.lock

javitonino opened the pull request.

mayoral commented:
Did you updated it from bundler 1.15.x? I only saw that that rack update when I was using the most recent version of bundler instead of the 1.14.x we have in the lockfile.

javitonino commented:
Nope, this is with 1.14.6. My guess is that we updated thin to 1.7 in one of the Rails 4 PRs, and in that one I manually tweaked the Gemfile (as I extracted it from a bigger PR), so I might have done this wrong.

In any case, rack doesn't seem to be updated here, only the requirement from thin. Rack itself [is still in version 1.4.7](https://github.com/CartoDB/cartodb/blob/b726edb4973120693aa87a4cb2a7dbe36977492d/Gemfile.lock#L196)

This however, changes the daemons update we discussed previously, but we already looked at the PR and it looked safe. Still, we should test this in staging (just a quick test to ensure it deploys fine).
